nbs/030_session.ipynb

```diff
@@ -54,9 +54,9 @@
     "\n",
```

Comment:
<p>you can use this function as a helper function to help make import_rel more readable, but there is no need to extend the public interface if import_rel already supports getting a Path to a csv as input</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spannerlib/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Changed it to _import_csv to not export it.</p><p> </p>